tanstack-table/src/main/scala/lucuma/react/virtual/VirtualOptions.scala

```diff
@@ -148,22 +148,34 @@ case class VirtualOptions[TScrollElement <: Element, TItemElement <: Element](
     p.count = count
     p.getScrollElement = () => getScrollElement.map(_.orUndefined).runNow()
     p.estimateSize = estimateSize
-    p.debug.foreach(v => p.debug = v)
```

Comment:
Heh. I guess these didn't do much before, eh?

Reply:
you won't believe how much it took me to realize this

Reply:
It `looks` just fine. 😆